src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```
@@ -3541,6 +3544,14 @@ compiler.err.module.not.found=\
 compiler.warn.module.not.found=\
     module not found: {0}
 
+# 0: name
+compiler.err.import.module.not.found=\
+    imported module not found: {0}
+
+# 0: symbol
+compiler.err.import.module.does.not.read=\
+    current modules does not read: {0}
```

```suggestion
    current module does not read: {0}
```

It would even better to print the name of the "current" module, so that the user-facing message reads:
`current module foo does not read module bar - possible fix: add "requires bar;" to foo's module descriptor `

---

Thanks. I've tweaked the error to include the module name here:
https://github.com/openjdk/jdk/pull/18614/commits/7cfaff80eac6261b62c5cdb9614a0e708bce7e33

Regarding possible fix(es) - it may be better to introduce some framework for that, and convert the most typical errors, separately. (As I don't think we do that for other errors.)